### DISCUSSION TITLE
examples folder doesn't have any docs

### DISCUSSION THREAD

**Discussion by jcollum-nutrien** [dotansimha/graphql-code-generator#8540] [Docs Discussions]
I look in the folder, no readme. At the top level the examples aren't discussed. I feel like I'm supposed to just poke around in the code and see if any of it might be what I need. That's not very newbie-friendly.

**Comment by charlypoly** [comment]
Hi @jcollum-nutrien,

Most examples are mentioned from documentation pages:
- `examples/typecript-esm` ➡️  https://www.the-guild.dev/graphql/codegen/docs/getting-started/esm-typescript-usage
- `examples/front-end` ➡️ https://www.the-guild.dev/graphql/codegen/docs/guides/react-vue
- `examples/programmatic-usage` ➡️ https://www.the-guild.dev/graphql/codegen/docs/advanced/programmatic-usage

I'll add a `REAME.md` in `examples/`
